docs/react/guides/filters.md

```diff
@@ -3,7 +3,7 @@ id: filters
 title: Filters
 ---
 
-Some methods within React Query accept a `QueryFilters` or `MutationFilters` object.
+Some methods within Tanstack Query accept a `QueryFilters` or `MutationFilters` object.
```

Comment:
should it be `TanStack` Query (with an upper case S) everywhere?

Reply:
Done ✅